rpcore/common_resources.py

```diff
@@ -187,7 +187,7 @@ def update(self):
         curr_vp = self._input_ubo.get_input("view_proj_mat_no_jitter")
         update("last_view_proj_mat_no_jitter", curr_vp)
         curr_vp = Mat4(curr_vp)
-        curr_vp.invert_in_place()
+        # curr_vp.invert_in_place()
```

Comment:
Is there a reason this belongs to the PR? Is this a bug? If so I'd appreciate if you'd split it into a second PR

Reply:
Yeah. It's a bug and don't work in latest Panda3D. I don't known if it worked before.

It breaks AO with clip length > 1, but at least it doesn't crashes the RP. Unfortunately I don't have a better solution.

Reply:
I don't think we can merge this code though. It would require some more testing since I think this matrix is used in many places. Are you sure the underlying issue is the inversion?

Reply:
I have tested again and I don't have the problem anymore. I have removed this.